libs/sdk-ui-dashboard/src/model/commands/filters.ts

```diff
@@ -339,16 +394,27 @@ export interface ChangeAttributeFilterSelection extends IDashboardCommand {
  * The attribute elements can be provided either using their URI (primary key) or value. Together with the
  * elements you must indicate the selection type - either 'IN' or 'NOT_IN'.
  *
+ * To convert {@link IDashboardFilter} to {@link @gooddata/sdk-model#IFilter} use {@link dashboardAttributeFilterToAttributeFilter}.
```

Comment:
Maybe we should suggest using the applyDateFilter instead to make it easier for the user?

Reply:
And remove the guide about the converting etc. since they have a better alternative now?